numbat/tests/interpreter.rs

```diff
@@ -227,6 +227,19 @@ fn test_function_inverses() {
 
 #[test]
 fn test_math() {
+    expect_output("solve_quadratic_equation(1, 0, -1)", "x₁ = 1; x₂ = -1");
+    expect_output("solve_quadratic_equation(9.0, -126.0, 441.0)", "x = 7");
+    expect_output("solve_quadratic_equation(9.0, -126.0, 441)", "x = 7");
+    expect_output("solve_quadratic_equation(9.0, -126, 441)", "x = 7");
+    expect_output("solve_quadratic_equation(9, -126, 441)", "x = 7");
```

Comment:
There's no point in having all these four lines, they're the exact same (Numbat doesn't have an integer type):

```suggestion
    expect_output("solve_quadratic_equation(9, -126, 441)", "x = 7");
```

Sorry for all the bikeshedding and nitpicking, almost there!